modules/openapi-generator/src/main/resources/go/client.mustache

```diff
@@ -138,6 +138,10 @@ func parameterToString(obj interface{}, collectionFormat string) string {
 		delimiter = "\t"
 	case "csv":
 		delimiter = ","
+	case "json": {
+	    jsonBuf, _ := json.Marshal(obj)
```

Comment:
If checking the `err` here will result in better error handling (e.g. users get notified of the errors instead of errors being silently discarded), then please do so.

Reply:
The function may need to be altered to return the error but it would be a good idea.

Reply:
Ok. I will add the error handling.

And since this is the only case that needs error handling (and this function was already being distorted anyway), I will also move JSON serialization code to the point of call instead of changing `parameterToString`.

Reply:
I've updated the code to add error handling. Please let me know what you think.

Reply:
@antihax can you please have another look when you've time?